Framework/Algorithms/src/FitPeaks.cpp

```diff
@@ -854,7 +854,9 @@ FitPeaks::fitPeaks() {
 namespace {
 /// Supported peak profiles for observation
 std::vector<std::string> supported_peak_profiles{"Gaussian", "Lorentzian",
-                                                 "PseudoVoigt", "Voigt"};
+                                                 "962"
```

Comment:
What peak profile is `962`?

Reply:
it is a mistype (not even a typo).  I will have it removed.